scss/elements/_sections.scss

```diff
@@ -1,308 +1,306 @@
+                  padding-left: ($col * 2);
+            }
+
+            li {
+                margin: 0 0 $baseline 0;
+                line-height: 32px;
+                font-weight: 400;
+                font-size: 18px;
+                padding: 0;
+                @include breakpoint(sm) {
+                    font-size: 16px;
+                }
+            }
+
+            a {
+                text-decoration: underline;
+                font-size: 18px;
+                font-weight: 400;
+                line-height: 32px;
+                @include breakpoint(sm) {
+                    font-size: 16px;
+                }
+              }
+
+            h1, h2 {
+                font-size: 30px;
+                margin: ($baseline * 4) 0 ($baseline * 4) 0;
+                padding: ($baseline * 3) 0 0 0;
+                font-weight: bold;
+                line-height: (40px);
+                @include breakpoint(sm) {
+                    font-size: 24px;
+                    margin: ($baseline * 3) 0 ($baseline * 2) 0;
+                    line-height: 32px;
+                }
+            }
+
+            h3 {
+                font-size: 24px;
+                margin: ($baseline * 4) 0 ($baseline * 2) 0;
+                padding: ($baseline * 3) 0 0 0;
+                font-weight: bold;
+                line-height: 32px;
+                @include breakpoint(sm) {
+                    font-size: 18px;
+                    line-height: 24px;
+                    margin: ($baseline * 3) 0 ($baseline * 2) 0;
+                }
+            }
+
+            h4 {
+                font-size: 18px;
+                margin: ($baseline *3) 0 ($baseline * 2) 0;
+                padding: ($baseline * 3) 0 0 0;
+                font-weight: bold;
+                line-height: 32px;
+                @include breakpoint(sm) {
+                    font-size: 16px;
+                    line-height: 24px;
+                    margin: ($baseline * 2) 0 ($baseline * 2) 0;
+                }
+            }
+
+            h5, h6 {
+                    font-size: 16px;
+                    margin: ($baseline * 2) 0 ($baseline) 0;
+                    @include breakpoint(sm) {
+                        margin: ($baseline * 2) 0 ($baseline * 2) 0;
+                    }
+            }
+
+            // All 1-4 headings should be bold. They should have padding above, 
+            // but only if not under a heading equal or lower level to itself
+            @for $i from 1 through 4 {
+                @for $j from $i through 6 {
+                    h#{$i} + h#{$j} {
+                        padding-top: 0;
+                   }
+                }
+            }
+
+            // Special instances of headings
+
+            // Sub headings
+            header + h3  {
+                padding-top: 0;
+            }
+
+            // Chart subtitles
+            h4 + h5  {
+                margin: 0 0 ($baseline * 2) 0;
+                font-weight: normal;
```

Comment:
Think I missed this, but h5 should `font-weight: normal` even when not proceeded with an h4. At the moment 'source' on tables and charts are bold when they should not be.